### PR TITLE
Unittests conversions

### PR DESCRIPTION
**Pull request by chfast**

### CODE REVIEW

**Comment by winsvega** [comment]
Please dont convert BasicTests/difficulty Tests.  (Looks like this pr not touching it)

**Comment by chfast** [comment]
Hey @winsvega,
I'm not going to convert any tests touching JSON tests. But I plan to move them to `tools/jsontests` as I did with RPCTests: https://github.com/ethereum/aleth/pull/5348/files#diff-871e2329bd9d4dfc77ee5c4ae7867f9cR52.

**Comment by gumb0** [comment]
Where did `ethashEvalHeader` go?

File: test/unittests/libethcore/KeyManager.cpp [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+
+    EXPECT_NO_THROW(km.create(password));
+    EXPECT_TRUE(km.accounts().empty());
+    EXPECT_TRUE(km.load(password));
+
+    for (auto a : km.accounts())
+        km.kill(a);
+}
+
+TEST(KeyManager, keyManagerKill)
```
Comment (halfalicious) [inline test/unittests/libethcore/KeyManager.cpp]: Nice job fixing the whitespaces in this test: https://github.com/ethereum/aleth/blob/9415843f53d0961c88081b3a6362c313bbbeb220/test/unittests/libethcore/KeyManager.cpp#L117-L135

**Pull request merged**